### DISCUSSION TITLE
Unable to mark multiple PRs as "closed"

### DISCUSSION THREAD

**Discussion by StaphSynth** [community/community#58218] [Pull Requests]
### Select Topic Area

Bug

### Body

I'm trying to close multiple PRs in a repository. If I go to one and close it, it's fine. However, if I select multiple PRs from the list and use the "mark as closed" option, it fails and I get a 422 from the server.

<img width="1302" alt="Screen Shot 2023-06-16 at 3 02 28 pm" src="https://github.com/community/community/assets/20519886/fd2fe892-abac-4e18-aa49-aa42b8d0a087">

**Comment by aahouzi** [comment]
Receiving a 422 error when trying to close multiple pull requests in a repository suggests that there might be an issue with the request being sent or the data being provided. The error code indicates that the server understands the request but cannot process it due to semantic errors.

Here are a few potential reasons and troubleshooting steps you can try:

1. Verify permissions: Make sure you have the necessary permissions to close pull requests in the repository. Ensure that you have the appropriate access level (e.g., write access) to modify pull requests.

2. Check pull request status: Ensure that the pull requests you are trying to close are in an open state. Closed pull requests cannot be closed again.

3. Review API rate limits: GitHub has rate limits in place for API requests. If you are making a large number of API requests within a short period, you might be hitting the rate limit. Wait for a moment and try again, or consider reducing the number of pull requests you're attempting to close in a single request.

4. Split the requests: Instead of selecting and closing multiple pull requests at once, try closing them individually or in smaller batches to see if that resolves the issue. This can help identify if there's a specific pull request causing the problem.

5. Clear browser cache: Clearing your browser cache and refreshing the page might resolve any potential caching issues that could interfere with the closing of multiple pull requests.

I hope it can help you somehow!
